conf/mpcdf.config

```
@@ -61,7 +61,7 @@ profiles {
        
         params {
             config_profile_description = 'MPCDF raven profile (unofficially) provided by nf-core/configs.'
-            max_memory = 2000000.MB
+            max_memory = 2048.GB
```

```suggestion
            max_memory = 2000000.MB
```

Changing to MB was a suggestion Mike got so will stick with that